resources/web/wwi/Parser.js

```diff
@@ -180,7 +219,16 @@ export default class Parser {
         result = this.#parseFog(node);
       else
         console.error('This world already has a fog.');
-    } else {
+    } else if (node.tagName === 'Transform' || node.tagName === 'Robot' || node.tagName === 'Solid' ||
+      node.tagName === 'Accelerometer' || node.tagName === 'Altimeter' || node.tagName === 'Camera' ||
+      node.tagName === 'Charger' || node.tagName === 'Compass' || node.tagName === 'Connector' || node.tagName === 'Display' ||
+      node.tagName === 'DistanceSensor' || node.tagName === 'Emitter' || node.tagName === 'GPS' || node.tagName === 'Gyro' ||
+      node.tagName === 'InertialUnit' || node.tagName === 'LED' || node.tagName === 'Lidar' ||
+      node.tagName === 'LightSensor' || node.tagName === 'Pen' || node.tagName === 'Radar' || node.tagName === 'RangeFinder' ||
+      node.tagName === 'Receiver' || node.tagName === 'Speaker' || node.tagName === 'TouchSensor' || node.tagName === 'Track' ||
+      node.tagName === 'TrackWheel')
+      result = this.#parseTransform(node, parentNode, isBoundingObject);
```

Comment:
I'm not sure if it's actually better, but since this if is getting kinda unwieldy, doing it with as a switch wouldn't be more clear? Like in this case:

```
switch(node.tagName) {
   ....
   case 'Accelerometer':
   case 'Altimeter':
   case 'Camera':
   <all others>
   case 'TrackWheel':
      result = this.#parseTransform(node, parentNode, isBoundingObject);

``

Reply:
Good idea